index.d.ts

```diff
@@ -79,7 +79,8 @@ export declare class GCSDataStore extends DataStore {
  * file store in AWS S3
  */
 export declare class S3Store extends DataStore {
-    constructor(options: S3StoreOptions);
+    constructor(options: S3StoreOptions & { accessKeyId: string, secretAccessKey: string });
+    constructor(options: S3StoreOptions & { credentials: ECSCredentials });
```

Comment:
If we allow all `credential` types, then the specific type of `ECSCredentials` is not what we want here. I think it might be the `RemoteCredentials` type?

Reply:
Yeah you're right
Or perhaps use the `AWS.Credentials` itself